main.js

```diff
@@ -1 +1,11 @@
+//task 1
+const currentDate = new Date();
+const hours = currentDate.getHours()<10? `0${currentDate.getHours()}` : currentDate.getHours();
+const minutes = currentDate.getMinutes()<10? `0${currentDate.getMinutes()}` : currentDate.getMinutes();
+const day = currentDate.getDate()<10? `0${currentDate.getDate()}` : currentDate.getDate();
+const month = currentDate.getMonth()<10? `0${currentDate.getMonth()}` : currentDate.getMonth();
```

Comment:
please, make pretty format for your code. Before and after `<` `?` should be used spaces